test_scripts/Smoke/Registration/ATF_Register_App.lua

```diff
@@ -0,0 +1,78 @@
+--  Requirement summary:
+--  [RegisterAppInterface] SUCCESS
+--  [RegisterAppInterface] RegisterAppInterface and HMILevel
+--
+--  Description:
+--  Check that it is able to register App.
+--
+--  1. Used precondition
+--  SDL, HMI are running on system.
+--  Mobile device is connected to system.
+--
+--  2. Performed steps
+--  appID->RegisterAppInterface(params)
+--
+--  Expected behavior:
+--  1. SDL successfully registers application and notifies HMI and mobile
+--     SDL->HMI: OnAppRegistered(params)
+--     SDL->appID: SUCCESS, success:"true":RegisterAppInterface()
+--  3. SDL assignes HMILevel after application registering:
+--     SDL->appID: OnHMIStatus(HMlLevel, audioStreamingState, systemContext)
+
+-- [[ Required Shared Libraries ]]
+local commonFunctions = require('user_modules/shared_testcases/commonFunctions')
+local commonSteps = require('user_modules/shared_testcases/commonSteps')
+local mobile_session = require('mobile_session')
+
+--[[ General Settings for configuration ]]
+Test = require('user_modules/dummy_connecttest')
+require('cardinalities')
+require('user_modules/AppTypes')
+
+-- [[Local variables]]
+local default_app_params = config.application1.registerAppInterfaceParams
+
+--[[ Preconditions ]]
+commonFunctions:newTestCasesGroup("Preconditions")
+commonSteps:DeletePolicyTable()
+commonSteps:DeleteLogsFiles()
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Check that it is able to register App")
```

Comment:
@LitvinenkoIra, Please correct description in following way (according to Atomic_template): 
```
commonFunctions:newTestCasesGroup("Test")
```